dials/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
@daonb please add a functional test for the view.. we don't add any more code to Open Knesset without tests. the era has ended..